cockroach-admin/api/src/lib.rs

```diff
@@ -12,6 +15,19 @@ use serde::{Deserialize, Serialize};
 pub trait CockroachAdminApi {
     type Context;
 
+    /// Initialize the CockroachDB cluster.
+    ///
+    /// This performs both the base-level `cockroach init` and installs the
+    /// Omicron schema. It should be idempotent, but we haven't heavily tested
+    /// that; in practice, only RSS calls this endpoint and does so serially.
```

Comment:
Aren't the tests below in `cockroach-admin/src/cockroach_cli.rs` testing idempotency here? We call cluster/schema initialization back-to-back?

Reply:
They are, but I think I'll stick with "not heavily tested", because we're not testing:

* What happens if this endpoint is called concurrently
* What happens if this endpoint is called concurrently on different cockroach nodes

because we don't actually do that in practice. (I think it would probably work, but relies on `cockroach init` itself being idempotent, which seems like a hard thing to know for sure?)

I could add these details to this comment?

Reply:
Yeah, the concurrency concern seems valid! The updated comment would help.